tutorials/pubsub-spring-dedup-messages/index.md

```diff
@@ -212,22 +217,22 @@ In [DedupPubSub.java](pubsubio-dedup/src/main/java/com/google/example/DedupPubSu
 
 ### Run the Dataflow job
 
-To start the Dataflow job,  navigate to `pubsubio-dedup/` and run: 
-```shell script
-  mvn compile exec:java \
-   -Dexec.mainClass=com.google.example.DedupPubSub \
-   -Dexec.cleanupDaemonThreads=false \
-   -Dexec.args="\
-     --project=$PROJECT_NAME \
-     --inputTopic=projects/$PROJECT_NAME/topics/topicToDataflow \
-     --outputTopic=projects/$PROJECT_NAME/topics/topicFromDataflow \
-     --idAttribute=key \
-     --runner=DataflowRunner"
-```
-You can check the job's progress in the [Cloud Console for Dataflow]. Wait a few minutes for the job status to become "running". Then issue `Ctrl+C` to stop the program locally. Stopping the program locally does not affect the Dataflow job running in the Cloud.
+1. To start the Dataflow job,  navigate to `pubsubio-dedup/` and run: 
+    ```shell script
+      mvn compile exec:java \
+       -Dexec.mainClass=com.google.example.DedupPubSub \
+       -Dexec.args="\
+         --project=$PROJECT_NAME \
+         --inputTopic=projects/$PROJECT_NAME/topics/topicToDataflow \
+         --outputTopic=projects/$PROJECT_NAME/topics/topicFromDataflow \
+         --idAttribute=key \
+         --runner=DataflowRunner"
+    ```
+1. Check the job's progress in the [Cloud Console for Dataflow]. Wait a few minutes for the job status to become "running". 
+1. Issue `Ctrl+C` to stop the program locally. Stopping the program locally does not affect the Dataflow job running on GCP.
 
 ## Observe the Results
-Publish a few messages of different keys via the web host and observe deduplicated messages in your terminal.
+Publish a few more messages of different keys (i.e. `idAttributes`) via the web host and observe messages of the same key arrive only once in your terminal.
```

Comment:
Can you add examples of key-message pairs and the expected output? Maybe something like:

```
key: "111", message: "hi"     # processed since key 111 has not been seen before
key: "222", message: "hello"  # processed
key: "111", message: "ignore" # ignored since key 111 has been seen before
```

Reply:
I added a paragraph beneath a screenshot of the output to describe the expected output.